core/lib/spree/testing_support/factories/address_factory.rb

```diff
@@ -11,7 +11,7 @@
     end
 
     firstname 'John'
-    lastname 'Doe'
+    lastname nil
```

Comment:
I hope it's true. We don't follow that strictly but I think it makes for more robust tests if we try to make factories minimal by default.  E.g. for this `lastname` scenario exactly -- you end up writing specs around attributes that the default factories provide but that aren't actually guaranteed to be there.  It's easy to add additional factories that are more "filled out", but I think it's nice to make that explicit.  Apps can also use `FactoryGirl.modify` if they'd like to have lastnames populated by default.